pkg/features/kube_features.go

```diff
@@ -913,6 +919,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	DisableCloudProviders:                          {Default: false, PreRelease: featuregate.Alpha},
 	StatefulSetMinReadySeconds:                     {Default: false, PreRelease: featuregate.Alpha},
 	ExpandedDNSConfig:                              {Default: false, PreRelease: featuregate.Alpha},
+	PodPriorityBasedGracefulShutdown:               {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
I might prefer this to be `GracefulNodeShutdownBasedOnPodPriority` so that it is next to GracefulNodeShutdown

Reply:
@smarterclayton
This is written in accordance with KEP https://github.com/kubernetes/enhancements/issues/2712, I don’t have any strong opinions, maybe we can go to KEP to discuss.
PTAL @mrunalp

Reply:
+1 for GracefulNodeShutdownBasedOnPodPriority
So we can find this feaute gate right next to GracefulNodeShutdown.

as this is `node graceful shutdown` feature and we should mention that this will take no effect if  GracefulNodeShutdown feature gate is disabled.

Reply:
@wzshiming Yeah, let's update it to GracefulNodeShutdownBasedOnPodPriority. We can update the KEP as well.